.github/workflows/ci.yaml

```diff
@@ -28,7 +28,7 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-node@v1
         with:
-          node-version: 12
+          node-version: 22
       - run: npm install
         env:
           PUPPETEER_SKIP_CHROMIUM_DOWNLOAD: true
```

Comment:
With the latest version this is now:
```suggestion
          PUPPETEER_SKIP_DOWNLOAD: true
```

Reply:
Thanks.

I found the PR where the change landed: https://github.com/puppeteer/puppeteer/pull/10054

> removes `PUPPETEER_SKIP_CHROMIUM_DOWNLOAD` in favor of `PUPPETEER_SKIP_DOWNLOAD` since the flag always affected any browser, not only Chromium.